sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md

```diff
@@ -23,7 +24,11 @@ Add-SPOListDesign -Title <String> -SiteScripts <SPOSiteScriptPipeBind[]> [-Descr
 ```
 
 ## DESCRIPTION
-Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+Creates a new list or document library design available to users when they create a new list or document library from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+
+List designs will be available in UI where lists are created, and document library designs will be available in UI where document librraries are created.  The difference is based on the "templateType" of the site design used.  "templateType" 100 is for Lists, "templateType" 101 is for document libraries.
```

Comment:
>"templateType" [](http://example.com/codeflow?start=186&length=14)

I don't see this param in the cmdlet; where does this come from?

Reply:
They're part of the xml definition of the list design.